controllers/utils/utils_test.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utils_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	// . "github.com/onsi/ginkgo/extensions/table"
+	// . "github.com/onsi/gomega"
```

Comment:
nit: remove unused imports. And what's the plan for this file? Will it be done in this PR? Or separately?